s96at/include/s96at.h

```diff
@@ -389,6 +389,34 @@ uint8_t s96at_lock_zone(struct s96at_desc *desc, enum s96at_zone zone, uint16_t
  */
 uint8_t s96at_read(struct s96at_desc *desc, enum s96at_zone zone, uint8_t id, uint8_t *buf);
 
+/* Put the device into the sleep state
```

Comment:
I haven't checked, but with checkpatch here, won't that complain about this comment? I believe checkpatch would like to see comments like this:
```c
/* This is a single line comment */

/*
 * And this is a multiline
 * comment
 */
```
I.e., in multiline comments the first line is simply just `/*`?

Reply:
The above is actually dictated by the kernel's coding style with the exception of networking code (https://www.kernel.org/doc/html/v4.10/process/coding-style.html#commenting), but for one reason or another checkpatch does not complain about it. Speaking of which, I updated my git hooks to run checkpatch before commit to make sure all future commits comply.

Reply:
I see, didn't realize that was exceptions for networking etc, well I'm fine with this too. Having checkpatch in the git hooks is a good idea, I used to have that a long time ago, I should probably re-enable that again.

Side note, I'm about to merge most of you open PRs today.